desktop/src/player.rs

```diff
@@ -259,6 +264,18 @@ impl ActivePlayer {
             }
         }
 
+        let gamemode_enable = match preferences.gamemode_preference() {
+            GameModePreference::Default => {
+                preferences.graphics_power_preference() == PowerPreference::High
+            }
+            GameModePreference::On => true,
+            GameModePreference::Off => false,
+        };
+
+        if cfg!(not(target_os = "linux")) && gamemode_enable {
+            tracing::warn!("Cannot enable GameMode, as it is supported only on Linux");
```

Comment:
Now that I look at it, this ends up adding a pointless warning on windows by default :(

Reply:
https://github.com/ruffle-rs/ruffle/pull/18242